ModelDerivative_01/instructions/before_you_begin.md

```diff
@@ -8,19 +8,17 @@ Postman environments are named configurations that implement environment variabl
 
 To import the Postman Collection and environment you need for this tutorial:
 
-1. Download the following files from the [*collections* folder](../collections).
-
-    1. *Model Derivative.postman_collection.json*
-    2. *Model_Derivative.postman_environment.json*
+1. Download the following zip file from the [*collections* folder](../collections) and extract it on your local machine.
 
+    - _model_derivative_collections.zip_
 
 2. In the Postman header bar, click **Import**. A dialog displays.
 
-3. Drag the file you downloaded in step 1 to the area marked **Drop files here**. Alternatively, you can click **Choose Files** and pick the files you downloaded in step 1.
+3. Drag the file you extracted in step 1 to the area marked **Drop files here**. Alternatively, you can click **Choose Files** and pick the files you downloaded in step 1.
```

Comment:
Please use the term "expanded" instead of "extracted". The Mac right-click menu refers to this operation as expand.

Reply:
Accepted and changed